gandalf.js

```diff
@@ -1,43 +1,43 @@
-const objectValues = Object.values(object);
```

Comment:
SCRIPT-I'm a bit concerned about the potential performance impact of this loop. Have you thought about optimizing it?